pkg/tracing/attributes.go

```diff
@@ -28,6 +28,10 @@ const (
 	MessagingDestinationAttributeName = "messaging.destination"
 	MessagingProtocolAttributeName    = "messaging.protocol"
 	MessagingMessageIDAttributeName   = "messaging.message_id"
+
+	K8sNamespaceName = "k8s.namespace"
+	K8sNameName      = "k8s.name"
+	K8sResourceName  = "k8s.resource" // Qualified resource name, eg. pingsource.sources.knative.dev
```

Comment:
The [k8s semantic convention](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/k8s.md) uses:
`k8s.<resource>.name` for the name, so should we use a similar thing instead?

`k8s.pingsource.sources.knative.dev.name` (like `k8s.deployment.name`)

For the namespace, it should be:
`k8s.namespace.name`

Reply:
> > Since OpenTelemetry does not support custom resources (yet), I chose what I think are reasonable span attribute names.
> 
> Should we propose one before we move too far down our custom names?

Yes I'm planning to. 

I was hesitant about using `k8s.<resource>.name` but it fits well into the current proposal, so +1. +1 as well for `k8s.namespace.name`. 

Thanks!

Reply:
fixed.